modules/apps/headless/headless-admin-list-type/headless-admin-list-type-impl/src/main/java/com/liferay/headless/admin/list/type/internal/resource/v1_0/ListTypeDefinitionResourceImpl.java

```diff
@@ -285,6 +291,33 @@ private ListTypeDefinition _toListTypeDefinition(
 				setName_i18n(
 					() -> LocalizedMapUtil.getI18nMap(
 						serviceBuilderListTypeDefinition.getNameMap()));
+				setPermissions(
+					() -> NestedFieldsSupplier.supply(
+						"permissions",
+						nestedFieldNames -> {
+							String permissionName =
+								com.liferay.list.type.model.ListTypeDefinition.
+									class.getName();
```

Comment:
Can you declare this variable before and replace for all usages of `com.liferay.list.type.model.ListTypeDefinition.class.getName()` ?